DSharpPlus/Entities/DiscordUser.cs

```diff
@@ -194,6 +194,9 @@ public string GetAvatarUrl(ImageFormat fmt, ushort size = 1024)
                     sfmt = "webp";
                     break;
 
+                case ImageFormat.Auto:
+                    sfmt = !string.IsNullOrWhiteSpace(this.AvatarHash) ? (AvatarHash.StartsWith("a_") ? "gif" : "png") : "png";
```

Comment:
Be consistent and add `this` the the second `AvatarHash`

Reply:
It was like this for AvatarUrl too. Thought that has some kind of purpose. Should I change it then there too?